fuels-abigen-macro/tests/test_projects/complex_types_contract/Forc.toml

```diff
@@ -0,0 +1,9 @@
+[project]
+author  = "Rodrigo Araujo"
+license = "MIT"
+name = "contract_test"
+entry = "main.sw"
+
+[dependencies]
+std = { path = "../lib-std" }
+core = { path = "../lib-core" }
```

Comment:
In a separate PR, can you remove the local version of these and use the versions on GitHub instead?